sycl/include/sycl/ext/oneapi/properties/property.hpp

```diff
@@ -169,7 +169,10 @@ enum PropKind : uint32_t {
   ImplementInCSR = 3,
   LatencyAnchorID = 4,
   LatencyConstraint = 5,
-  PropKindSize = 6,
+  WorkGroupSize = 6,
+  WorkGroupSizeHint = 7,
+  SubGroupSize = 8,
+  PropKindSize = 9,
```

Comment:
is this correct? changing `PropKindSize` from 6 now to 9? Won't that lead to incompatibilities with things built against older libs.

Reply:
The `PropKindSize` is intended to move, to give an upper bound on the property kind numbering. At the moment it is not used for anything other than cheating in unittests, but even if it was to be used for anything later it should only be to indicate the number of properties defined.